live-event/README.md

```diff
@@ -0,0 +1,20 @@
+# Live Event app
+
+`live-event` is a …
```

Comment:
add description, this is not a one on one app, but one to many? Or just call it group chat?